pkg/cmd/server/api/types.go

```diff
@@ -399,8 +399,10 @@ type MasterConfig struct {
 	// AuditConfig holds information related to auditing capabilities.
 	AuditConfig AuditConfig
 
-	// EnableTemplateServiceBroker is a temporary switch which enables TemplateServiceBroker.
-	EnableTemplateServiceBroker bool
+	// TemplateServiceBrokerConfig holds information related to the template
+	// service broker.  The broker is enabled if TemplateServiceBrokerConfig is
+	// non-nil.
+	TemplateServiceBrokerConfig *TemplateServiceBrokerConfig
```

Comment:
one of our option questions here was whether this should be nested into a generic "ServerBrokerConfigs" container struct so we can have multiple broker configuration objects in the future (for different brokers).

The current thinking is we're more likely to move this config *out* of the master, than introduce new brokers into the master, but I'd like @smarterclayton to confirm that view before we paint ourselves into a corner.